src/System.Reflection.Emit/pkg/System.Reflection.Emit.pkgproj

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <SkipValidatePackage>true</SkipValidatePackage>
```

Comment:
Curious what validation errors you came up with.  I would imagine that getting the shape of the package correct is the goal of this PR and that's what validation checks.

Reply:
With my latest set of changes I have the pkgproj working and almost passing validation. The last piece failing validation is:
```
F:\git\corefx\Tools\Packaging.targets(1131,5): error : System.Reflection.Emit should be supported on .NETStandard,Version=v1.1 but has no runtime assets. [F:\git\corefx\src\System.Reflection.Emit\pkg\System.Reflection.Emit.pkgproj]
F:\git\corefx\Tools\Packaging.targets(1131,5): error : System.Reflection.Emit should be supported on .NETStandard,Version=v1.2 but has no runtime assets. [F:\git\corefx\src\System.Reflection.Emit\pkg\System.Reflection.Emit.pkgproj]
```

And that makes sense, it is caused because in the latest package (4.3.0) we have a ref for netstandard1.1, but have an implementation for netstandard1.3. I guess that in order for these errors to go away, I would have to add a cross compilation of the assembly on netstandard1.1 which would be a turd assembly as well. I'm curious as to why did we ever shipped a package like that which seems pretty broken to me.

Reply:
That requirement was a new one we started following in 2.0.  Before that we relied on NuGet doing ref/lib matching to reject the package if it had a ref with no lib.  NuGet ended up removing that check and *relying* on ref with no lib as a valid encoding for shared framework.

We should add that not-supported assembly for ns1.1.  We should also double check if the ns1.3 impl should be moved to netcoreapp / uap (for instance if it is forwarding to our corelib).